www/src/data/sidebars/doc-links.yaml

```diff
@@ -305,8 +305,8 @@
               link: /docs/seo/
             - title: Optimizing Site Performance with Guess.js
               link: /docs/optimizing-site-performance-with-guessjs/
-        - title: Localizing Your Site*
-          link: /docs/localization-i18n/
+        - title: Internationalizing Your Site (i18n)
```

Comment:
Any feedback on this title change? The distinction between localization and internationalization is sorta unclear to me even after reading up on it. My hunch is that "internationalizing" doesn't make sense as a verb, but I was hesitant to make a switch to "localizing" in the doc. Happy to revisit this next week, if not sooner. https://www.w3.org/International/questions/qa-i18n